sql_user_check_looker/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
Check if jq and curl are present. Raise error accordingly

Reply:
done.